packages/protect/src/lib/protect.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ *
+ * Copyright © 2025 Ping Identity Corporation. All right reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ *
+ */
+
+import { ProtectConfig, Protect } from './protect.types.js';
+
+// Add Signals SDK namespace to the window object
+declare global {
+  interface Window {
+    _pingOneSignals: {
+      init: (initParams?: ProtectConfig) => Promise<void>;
+      getData: () => Promise<string>;
+      pauseBehavioralData: () => void;
+      resumeBehavioralData: () => void;
+    };
+  }
+}
+
+/**
+ * @async
+ * @function createProtect - returns a set of methods to interact with the PingOne Signals SDK
+ * @param {ProtectConfig} options - the configuration options for the PingOne Signals SDK
+ * @returns {Promise<Protect>} - a set of methods to interact with the PingOne Signals SDK
+ */
+export async function createProtect(options: ProtectConfig): Promise<Protect> {
```

Comment:
I'm torn on the `createProtect` name.  @ancheetah and @ryanbas21, if we use `create` as a prefix here, we are setting a precedent for our other modules. Some don't use a prefix at all. Here are some options:

- no prefix, so just `protect`
- `create<feature>`
- `init<feature>`
- `initialize<feature>`
- `return<feature>`

Sorry for being pedantic, but I just want to think though how this naming convention scales since we will have many modules that will have to adopt this convention.

Reply:
I like the first option (no prefix, just `protect`). Not a fan of `init` because the Protect/Signals SDK doesn't actually get initialized until you call start. On the other hand using it to imply a sort of 'initializing protect api' is confusing.